src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -15,6 +16,10 @@ const PAGINATION_MENU =
   'div[data-ouia-component-type="PF4/PaginationOptionsMenu"]';
 const TOOLBAR_CONTENT = '.pf-c-toolbar__content';
 const TOOLBAR = '.pf-c-toolbar';
+const DEFAULT_ROW_COUNT = 20;
```

Comment:
Nit-pick: if you want to keep this value in sync with the source code, better to import the default value from https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/src/Services/Filters.js#L4
```
import { AFFECTED_CLUSTERS_INITIAL_STATE } from '../../../Services/Filters.js'

...

const DEFAULT_ROW_COUNT = AFFECTED_CLUSTERS_INITIAL_STATE.limit;
```

Reply:
Nice, but I think we should do that only in case we are the ones deciding the actual value. If it is someone that will just say, "hey, use this value" rather than changing it, then I think it is better to have it in the test rather than importing. Is this value up to us, or are UX designers or PMs deciding it?

Reply:
Oh, yes, this makes sense, thank you, let's keep it as it is. I think this is a decision of UXs.

Reply:
Great, thank you!